projects/angular-tooltip/src/lib/tooltip.component.scss

```diff
@@ -1,5 +1,5 @@
 :host {
-  max-width: 200px;
+  max-width: 300px;
```

Comment:
This max-width define tooltip width for all users? If yes its better define a default value and make this a config parameter